### PR TITLE
Add support for minimum_jobs

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

File: jobqueue_features/tests/resources/test_base.py [hunk 1]
```diff
@@ -1,324 +1,354 @@
...
-                # Add a sleep to make the task substantial enough to require scaling
-                time.sleep(1)
-                return return_string
-
-            self._multi_mpi_tasks_assert(task)
-
-            controller._close()
-        else:
-            pass
+import os
```
Comment (ocaisa) [inline jobqueue_features/tests/resources/test_base.py:1]: Urgh, different line endings in this file to everywhere else

File: jobqueue_features/tests/resources/test_base.py [hunk 2]
```diff
@@ -1,324 +1,362 @@
...
+        self.executable = "python"
+        self.script_path = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), "helloworld.py")
+        )
+
+    def tearDown(self):
+        # Kill any existing clusters
+        controller._close()
+
+    def _test_minimum_jobs(self):
```
Comment (ocaisa) [inline jobqueue_features/tests/resources/test_base.py:173]: @AdamWlodarczyk This is the new test, the other stuff is just line endings

**Review by AdamWlodarczyk — APPROVED** [review]
LGTM

**Comment by ocaisa** [comment]
CI will pass for the tutorial on merge

**Pull request merged**